src/components/Common/AutoFocusInput.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <b-form-input
+    :ref="refName"
+    v-model="value"
+    v-bind="$attrs"
+    @blur="looseFocus"
+    data-auto-focus="true"
+  />
+</template>
+
+<script>
+export default {
+  name: 'AutoFocusInput',
+  props: {
+    name: {
+      type: String,
+      required: true
+    },
+    initialValue: {
+      type: String,
+      default: ''
+    }
+  },
+  computed: {
```

Comment:
NITP: Usually, we follow a convention for the order of the members of a Vue Single File Component:
* name
* components
* directives
* props
* data
* computed
* methods
* mounted
* watch
* (other lifecycle hooks)